python/lsst/ts/scheduler/utils/test/feature_scheduler_sim.py

```diff
@@ -0,0 +1,218 @@
+# This file is part of ts_scheduler
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["FeatureSchedulerSim"]
+
+import logging
+import types
+
+from astropy.time import Time
+
+from lsst.ts.scheduler.driver import FeatureScheduler
+
+from lsst.ts.observatory.model import ObservatoryModel
+from lsst.ts.observatory.model import ObservatoryState
+
+from lsst.ts.dateloc import ObservatoryLocation
+
+from lsst.ts.astrosky.model import AstronomicalSkyModel
+
+from rubin_sim.site_models.seeingModel import SeeingModel
+from rubin_sim.site_models.cloudModel import CloudModel
+from rubin_sim.site_models.downtimeModel import DowntimeModel
+
+
+class FeatureSchedulerSim:
+    """Utility class to simulate observations with the FeatureScheduler.
+
+    This class is used mostly for unit testing.
+    """
+
+    def __init__(self, log: logging.Logger) -> None:
+
+        self.log = log.getChild(__name__)
+
+        self.start_time = Time(59853.983, format="mjd", scale="tai")
+        # Step in time when there is no target (in seconds).
+        self.no_target_time_step = 120.0
+
+        self.raw_telemetry = dict()
```

Comment:
Are the keys of this dict described somewhere? If yes then please add a link to the description. If no then please describe here. Perhaps a dedicated class would be better?